x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/TokenServiceTests.java

```diff
@@ -187,189 +186,6 @@ public void testInvalidAuthorizationHeader() throws Exception {
         }
     }
 
-    public void testRotateKey() throws Exception {
```

Comment:
This code is still present in the token service, right? If so I don't think we should delete the tests until we remove the code